db_handler.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from contextlib import contextmanager
+
+Base = declarative_base()
+engine = create_engine('sqlite:///game_of_dbs.db', echo=True)
+Session = sessionmaker(bind=engine)
+
+
+def create_all_tables():
+    Base.metadata.create_all(engine)
+
+
+def recreate_all_tables():
```

Comment:
This method is relevant only for the tests, so it would look better as a fixture.

Reply:
I will call it from the fixture, Still leaving it in this class as it's the actual specific implementation which the test should not be aware of